remote-workstations/new-agent-vm/Install-Idle-Shutdown.sh

```diff
@@ -13,6 +19,16 @@ SERVICE_CONFIG="${SERVICE_CONFIG_PATH}/CAMIdleShutdown.conf"
 
 MONITOR_SCRIPT=/opt/Teradici_CAM_idle_shutdown.py
 
+# Idle time in minutes before auto-shutdown should be engaged
+IDLE_TIMER=240
```

Comment:
Any reason why change for 60 to 240 mins? That's 4 hours....

Reply:
@bill-dall and I talked about this a bit and felt that 60 minutes seemed too short...

Reply:
If will only shut down if there is no PCoIP session. If there is a PCoIP session, even if user is doing nothing, it won't get shut down.